File: 프로그래머스/1/12926. 시저 암호/시저 암호.py
def solution(s, n):
    answer = ''
    for str in s:
        if str.isupper():
            tmp = (ord(str) - ord('A') + n) % 26
            answer += chr(tmp + 65)
        elif str.islower():
            tmp = (ord(str) - ord('a') + n) % 26
            answer += chr(tmp + 97)
        else:
            answer += str
    return answer

File: 프로그래머스/1/12926. 시저 암호/시저 암호.java
class Solution {
    public String solution(String s, int n) {
        String answer = "";
        for(char c: s.toCharArray()) {
            int code = (int)c;
            if (c==' ') answer += c;
            else if (Character.isUpperCase(c)) {
                answer += (char)((c - 'A' + n) % 26 + 'A');
            }
            else {
                answer += (char)((c - 'a' + n) % 26 + 'a');
            }
        }
        return answer;
    }
}